python/Ganga/Core/MonitoringComponent/Local_GangaMC_Service.py

```diff
@@ -1162,6 +1162,8 @@ def credChecker():
             except Exception as msg:
                 return False
             else:
+                if s is None:
+                    return True
```

Comment:
This whole section could be
```python
try:
    credObj.renew()
except CredentialRenewalError:
    return False
else:
    return True
```
since `renew()` will never return a value any more.